src/main/java/uk/gov/hmcts/reform/managecase/api/controller/NoticeOfChangeController.java

```diff
@@ -182,4 +194,48 @@ private void validateCaseIds(String caseId) {
             throw new ValidationException("Case ID should contain digits only");
         }
     }
+
+    @PostMapping(path = REQUEST_NOTICE_OF_CHANGE_PATH, produces = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "Request Notice of Change Event", notes = "Request Notice of Change Event")
+    @ResponseStatus(HttpStatus.CREATED)
+    @ApiResponses({
+        @ApiResponse(
+            code = 201,
+            message = REQUEST_NOTICE_OF_CHANGE_STATUS_MESSAGE
+        ),
+        @ApiResponse(
+            code = 400,
+            message = "One or more of the following reasons:"
+                + "\n1) " + ValidationError.CASE_ID_INVALID
+                + "\n2) " + ValidationError.CASE_ID_INVALID_LENGTH
+                + "\n3) " + ValidationError.CASE_ID_EMPTY,
+            response = ApiError.class,
+            examples = @Example({
+                @ExampleProperty(
+                    value = "{\n"
+                        + "   \"status\": \"BAD_REQUEST\",\n"
+                        + "   \"message\": \"" + ValidationError.CASE_ID_EMPTY + "\",\n"
+                        + "   \"errors\": [ ]\n"
+                        + "}",
+                    mediaType = APPLICATION_JSON_VALUE)
+            })
+        ),
+        @ApiResponse(
+            code = 401,
+            message = AuthError.AUTHENTICATION_TOKEN_INVALID
+        ),
+        @ApiResponse(
+            code = 403,
+            message = AuthError.UNAUTHORISED_S2S_SERVICE
+        )
+    })
+    public RequestNoticeOfChangeResponse requestNoticeOfChange(@Valid @RequestBody RequestNoticeOfChangeRequest
+                                                                                        requestNoticeOfChangeRequest) {
+        VerifyNoCAnswersRequest verifyNoCAnswersRequest
+            = new VerifyNoCAnswersRequest(requestNoticeOfChangeRequest.getCaseId(),
+                                          requestNoticeOfChangeRequest.getAnswers());
+        NoCRequestDetails noCRequestDetails = verifyNoCAnswersService.verifyNoCAnswers(verifyNoCAnswersRequest);
+        return requestNoticeOfChangeService.requestNoticeOfChange(noCRequestDetails);
```

Comment:
It would be better to push these lines as a single method to service layer.

Reply:
I think it is better here as a separation of concerns, the `RequestNoticeOfChangeService` doesn't and shouldn't need to know anything about `VerifyNocAnswersService`.  It also makes mocking and testing a lot more complex